### DISCUSSION TITLE
Is it possible to disable generator on-demand?

### DISCUSSION THREAD

**Discussion by sneko** [prisma/prisma#16705] [Q&A] [answered]
Hi,

In my case I would like to use a specific generator based on the environment. From what I saw there is no if/else statements available in the `prisma.schema` file. I do see the usage of environment variables, but generators don't have a property "enable: 'true'|'false'" to make things happening.

Did any of you succeed in doing so?

Thank you,

**Comment by janpio** [comment, marked as answer]
We have an open PR that will enable this: https://github.com/prisma/prisma/pull/16452

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
